file1.txt

```diff
@@ -1 +1,2 @@
 some text
+some text added by nguthrie locally
```

Comment:
I like where you're going with this. But aren't you missing a couple more lines like this???